index.html

```diff
@@ -303,6 +303,15 @@ <h3 class="grouping-title">Methods</h3>
               current animation to the beginning, you should also set the
               currentTime property to 0.</p>
             </li>
+            <li>
+              <div>toDataURL(type, encoderOptions)</div>
+              <p>Returns a screenshot of the current model render in the format
+              specified by <i>type</i> (defaults to image/png). The screenshot is
+              encoded as a data URL string. In formats that support a sliding scale
+              of quality (such as image/jpeg and image/webp) you can also specify a
+              value for <i>encoderOptions</i> between 0 and 1 (<i>encoderOptions</i>
```

Comment:
If the only encoder option is quality, should we just call this quality?

Reply:
Yeah, it's kind of a bad name. It was chosen for us, though: https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toDataURL